samples/sensor/grow_r502a/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_LOG=n
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+CONFIG_SENSOR=y
+CONFIG_GROW_R502A=y
```

Comment:
`CONFIG_GROW_R502A=y` not needed will get set by Kconfig & dts automatically.

Reply:
fixed